build.gradle

```diff
@@ -62,6 +62,25 @@ subprojects {
     apply plugin: "maven-publish"
     apply plugin: "org.embulk.embulk-plugins"
 
+    test {
+        testLogging {
+            outputs.upToDateWhen { false }
+            showStandardStreams = true
+        }
+    }
```

Comment:
Just wanted to make sure that this is intended?

Reply:
Ah, it may not be for this PR's main goal. If needed, I'll move this out to another PR.

It's to make local test runs more friendly by:
* Running tests always when `./gradlew test` even if the source code is not updated from the last run.
* Showing test's standard output/erorr to the console so that it is easier to check the test logs.

Reply:
Looks good to me, no need for a different PR.